src/soundio/soundmanager.h

```diff
@@ -126,6 +125,9 @@ class SoundManager : public QObject {
     void closeDevices(bool sleepAfterClosing);
 
     void setJACKName() const;
+    bool jackApiUsed() const {
+        return m_jackSampleRate.isValid();
```

Comment:
Is this correct? `m_jackSampleRate` seems to be set in `SoundManager::queryDevicesPortaudio()` when the JACK API is available. So it sounds like it will also be set when Mixxx uses ALSA, WasAPI, CoreAudio, etc. while a JACK server is available. I have not yet checked if this is actually the case here.

Reply:
Ups, good catch. Fixed.